Editor/Modules/FogModeAnalyzer.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using Unity.ProjectAuditor.Editor.Core;
+using Unity.ProjectAuditor.Editor.Diagnostic;
+using UnityEditor;
+using UnityEngine.Rendering;
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    enum FogModeStripping
+    {
+        Automatic,
+        Custom
+    }
+
+    public enum FogMode
+    {
+        None,
```

Comment:
I don't think we need `None` as it's only used in test code.

Reply:
In this case, how do I handle the none situation?